### PR TITLE
Merge from shared changes

### PR DESCRIPTION
**Pull request by millerds**
Thank you for your pull request!  Please provide the following information.

---

**Change Description**:
Get common updates from other repository

1. **Do these changes impact any *npm scripts* commands (in package.json)?** (e.g., running 'npm run start')
No.

2. **Do these changes impact *VS Code debugging* options (launch.json)?**
No.

3. **Do these changes impact *template output*?** (e.g., add/remove file, update file location, update file contents)
No.

4. **Do these changes impact *documentation*?** (e.g., a tutorial on https://docs.microsoft.com/en-us/office/dev/add-ins/overview/office-add-ins)
No.

If you answered yes to any of these please do the following:
    > Include 'Rick-Kirkham' in the review
    > Make sure the README file is correct

**Validation/testing performed**:
Ran tests locally.

### CODE REVIEW

File: manifest.xml [hunk 1]
```diff
@@ -101,11 +101,11 @@
         <bt:String id="Functions.Namespace" DefaultValue="CONTOSO" />
         <bt:String id="GetStarted.Title" DefaultValue="Get started with your sample add-in!" />
         <bt:String id="CommandsGroup.Label" DefaultValue="Commands Group" />
-        <bt:String id="TaskpaneButton.Label" DefaultValue="Show Taskpane" />
+        <bt:String id="TaskpaneButton.Label" DefaultValue="Show Task Pane" />
       </bt:ShortStrings>
       <bt:LongStrings>
-        <bt:String id="GetStarted.Description" DefaultValue="Your sample add-in loaded successfully. Go to the HOME tab and click the 'Show Taskpane' button to get started." />
-        <bt:String id="TaskpaneButton.Tooltip" DefaultValue="Click to Show a Taskpane" />
+        <bt:String id="GetStarted.Description" DefaultValue="Your sample add-in loaded successsfully. Go to the HOME tab and click the 'Show Task Pane' button to get started." />
```
Comment (Rick-Kirkham) [inline manifest.xml:107]: Need to preserve the spelling fix Elizabeth made a while back.

```suggestion
        <bt:String id="GetStarted.Description" DefaultValue="Your sample add-in loaded successfully. Go to the HOME tab and click the 'Show Task Pane' button to get started." />
```

File: test/end-to-end/test-manifest-debugging.xml [hunk 2]
```diff
@@ -66,11 +66,11 @@
         <bt:String id="Functions.Namespace" DefaultValue="CONTOSO"/>
         <bt:String id="GetStarted.Title" DefaultValue="Get started with your sample add-in!"/>
         <bt:String id="CommandsGroup.Label" DefaultValue="Commands Group"/>
-        <bt:String id="TaskpaneButton.Label" DefaultValue="Show Taskpane"/>
+        <bt:String id="TaskpaneButton.Label" DefaultValue="Show Task Pane"/>
       </bt:ShortStrings>
       <bt:LongStrings>
-        <bt:String id="GetStarted.Description" DefaultValue="Your sample add-in loaded successfully. Go to the HOME tab and click the 'Show Taskpane' button to get started."/>
-        <bt:String id="TaskpaneButton.Tooltip" DefaultValue="Click to Show a Taskpane"/>
+        <bt:String id="GetStarted.Description" DefaultValue="Your sample add-in loaded successsfully. Go to the HOME tab and click the 'Show Task Pane' button to get started." />
```
Comment (Rick-Kirkham) [inline test/end-to-end/test-manifest-debugging.xml:72]:
```suggestion
        <bt:String id="GetStarted.Description" DefaultValue="Your sample add-in loaded successfully. Go to the HOME tab and click the 'Show Task Pane' button to get started." />
```

File: test/end-to-end/test-manifest.xml [hunk 3]
```diff
@@ -65,11 +65,11 @@
         <bt:String id="Functions.Namespace" DefaultValue="CONTOSO"/>
         <bt:String id="GetStarted.Title" DefaultValue="Get started with your sample add-in!"/>
         <bt:String id="CommandsGroup.Label" DefaultValue="Commands Group"/>
-        <bt:String id="TaskpaneButton.Label" DefaultValue="Show Taskpane"/>
+        <bt:String id="TaskpaneButton.Label" DefaultValue="Show Task Pane"/>
       </bt:ShortStrings>
       <bt:LongStrings>
-        <bt:String id="GetStarted.Description" DefaultValue="Your sample add-in loaded successfully. Go to the HOME tab and click the 'Show Taskpane' button to get started."/>
-        <bt:String id="TaskpaneButton.Tooltip" DefaultValue="Click to Show a Taskpane"/>
+        <bt:String id="Contoso.GetStarted.Description" DefaultValue="Your sample add-in loaded successsfully. Go to the HOME tab and click the 'Show Task Pane' button to get started." />
```
Comment (Rick-Kirkham) [inline test/end-to-end/test-manifest.xml:71]:
```suggestion
        <bt:String id="Contoso.GetStarted.Description" DefaultValue="Your sample add-in loaded successfully. Go to the HOME tab and click the 'Show Task Pane' button to get started." />
```

**Review by Rick-Kirkham — APPROVED** [review]

**Review by Rick-Kirkham — CHANGES_REQUESTED** [review]
See inline comments.

**Review by Rick-Kirkham — APPROVED** [review]

**Review by Oleg-O — APPROVED** [review]

**Pull request merged**